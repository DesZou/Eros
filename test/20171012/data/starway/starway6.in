287643 256614 1000
58912 113056
229992 155752
164414 143666
197634 206439
126859 198113
149491 107795
216359 152402
17306 111053
148080 35642
235298 158993
104479 41045
256861 16363
224611 248419
83770 9553
192313 227681
181110 150966
136863 225855
100597 192002
233809 228043
164480 241488
76474 109286
217376 248040
120261 187981
248523 94399
49964 214892
116213 137766
168123 54855
160394 254722
132581 10038
263206 72712
135249 152135
264042 18517
265224 170946
287391 132983
45361 76518
200462 186892
244587 16532
193580 65835
231540 58037
852 97007
206829 246532
35867 151423
212295 229772
58833 53294
249161 134714
26681 140199
150317 167013
24785 45239
140092 178541
279095 146305
252090 56614
2707 171708
249990 74654
64992 180704
14556 176022
261173 204168
106437 21394
223741 46886
43975 177639
148104 161825
245705 202004
228973 143733
215792 236894
88945 138857
127078 216683
228170 108661
268547 93599
36294 905
256545 122390
85901 216185
90975 183955
59456 45613
107894 254036
170204 213559
129861 186775
255901 237049
153632 52616
166536 119513
157749 137080
117064 233840
92031 57475
98560 17414
221314 21527
169387 57098
246787 222096
31656 161394
270768 68432
163527 129280
49359 204629
77823 161946
207679 227407
258210 191302
14913 12836
34498 112222
48154 201045
261028 188454
135507 91203
72720 136142
43281 143125
253035 68985
106039 207965
234306 37574
73183 153254
193234 108542
5051 50310
59225 125236
19019 199625
82185 190331
275918 125601
56826 175344
42157 221968
231124 70396
28971 132095
53563 137805
218909 237418
171884 122217
53415 246450
260157 418
192875 121074
89266 26444
278513 9548
212857 169725
133347 196562
71110 72652
27542 214020
173190 120926
155200 227967
58003 57490
130797 14156
14170 62407
192627 4645
59494 10750
228294 36624
70881 48118
115451 202922
241461 76747
127534 190111
242141 220304
136200 27418
151932 222299
275273 21454
115250 24702
90574 66644
157793 235911
26669 40201
29462 236317
37320 142051
150875 122397
36418 62194
266759 143035
17646 37508
186908 105957
73399 197455
149039 5258
69224 158871
99169 100820
98442 210844
221217 4842
131616 114953
170057 175952
277114 246953
108945 13764
277681 237385
95081 15032
250165 167185
231063 74558
248291 198415
9028 194269
98680 189738
44402 247184
114487 153865
136444 247934
150518 66761
265419 235506
165062 197116
92705 246644
173228 240207
75447 125610
114352 44516
239549 95440
99475 65684
121064 218015
204598 67787
120716 253886
177061 154409
233819 248627
7562 31655
48839 98023
143700 57760
198903 181554
158538 200949
241004 118595
118799 17570
279112 120398
234241 114317
252580 253160
100053 67061
61239 189028
205077 182153
226775 27314
199492 144778
82140 209941
240585 116527
227350 159343
196859 137641
148764 253079
157011 247957
63929 228861
215660 127367
177569 51128
233922 185917
175151 20296
196351 128823
106765 63893
261900 161432
71141 137317
48131 83278
44053 116753
114431 150215
207189 158599
99567 111228
228913 54244
96351 103654
31224 22126
33712 3371
2522 8565
214295 119330
103639 190107
178378 9380
253347 255581
221481 231869
165731 38747
280577 132428
180964 4675
243180 7538
68191 131259
97673 151254
18440 219522
31855 105026
109603 126065
31892 250506
105793 191664
176664 235936
279233 69440
154574 137097
68823 223881
170388 156644
286923 151527
160981 201421
124347 22995
227177 87647
98651 5160
249504 80427
184735 235100
63222 13589
103904 77448
136379 50139
156806 30579
221241 163156
30155 163840
153777 143462
129719 253971
29441 220546
38589 255171
165464 12086
235752 174617
164395 161084
39590 61342
257212 105488
31392 90503
221228 52048
210628 237228
128564 187460
43322 212790
131940 241576
137120 909
37046 236510
248801 253591
163413 253258
150728 126808
170841 62864
3167 6474
31060 83166
231024 48993
136350 206722
173116 107805
83277 232023
25253 191204
23609 156169
99113 227926
194020 81706
177163 223320
108473 144214
273795 116301
194868 44522
277125 43248
60876 52419
114818 196838
255258 215279
137616 40074
199094 150730
46411 249399
55440 42648
123103 119129
183785 61477
262306 171943
140894 100866
10051 183743
49102 212245
88275 184705
112514 2926
83293 137114
256079 6428
229036 41097
273478 159386
239686 98742
153288 116465
54917 3922
201898 48620
250168 74507
253711 87033
241247 137615
99938 73542
205100 219197
12240 181373
151215 140667
126928 94069
175257 65145
82588 201593
52673 202933
26725 238118
223525 123203
159095 23305
247663 25991
62892 243014
280923 102560
237426 227223
245978 58285
244940 197948
81308 121089
224827 74578
220446 9223
287451 241137
158267 88909
8072 12757
150464 131284
146599 153389
207068 134659
280198 74933
14757 155596
245384 90711
40245 157289
88154 130078
176861 152939
143973 93918
40296 159755
113091 222685
56198 197481
12059 2706
169186 89524
148881 34538
40737 6154
233435 195465
184023 61801
172408 230020
24993 175177
255250 155900
115388 167515
40125 89277
119212 195618
82879 116369
222488 46724
109172 142333
220160 204484
237080 95524
251352 83035
270859 141119
168049 51065
190483 179599
77259 186622
281865 147223
188408 95553
170980 50594
124152 225283
41401 120644
110790 18493
222652 78462
64236 240733
197613 175063
85551 77947
197451 208798
196559 148260
460 104157
76063 194455
274597 37667
76746 225365
214245 79385
29964 16482
91182 130221
187724 91630
90449 66224
66624 250624
163971 197439
44464 150312
246614 147954
215555 138886
54140 227511
133118 33503
236077 242350
182102 159272
166900 57651
217108 124796
426 34246
60286 213737
130426 232208
144298 205417
18463 36379
163955 22499
24361 86239
255346 233171
263158 27729
188439 9152
246953 122647
228161 196776
63149 111218
188475 132228
54457 254550
282234 197842
122456 95950
35615 119838
104424 50117
234171 171488
210935 128594
230769 174955
102815 237893
261403 110233
28185 122644
25203 20241
49577 231311
183825 154844
249853 102922
263482 42725
36703 38993
277684 104981
176400 212461
259727 208025
155505 116593
273595 40678
153818 159026
114119 14799
34789 224619
222456 241773
173205 79600
185761 19480
234119 158888
29361 48124
129312 34872
179217 126843
35930 43936
153640 21077
227913 158453
90989 141288
197171 187949
412 76989
219680 248332
165009 193916
155317 101584
9423 155976
9779 7854
234696 59163
267553 36758
237361 17003
62244 125372
14109 39059
123401 26511
224499 194583
201308 17090
150529 28032
162915 152906
239567 222758
144673 95974
254933 153684
163689 21908
10119 10312
58515 48044
235390 81704
153332 47535
265783 4369
139886 146256
129425 30188
187281 186548
81897 161207
84959 29716
261427 16651
39429 32517
105528 186629
244688 173753
69886 105003
50233 58333
169573 27122
174841 234006
57580 255321
272696 216811
207708 244285
88202 110263
237300 64341
179857 29424
226370 115302
226727 234080
73664 83507
255413 84149
168500 227028
218728 97559
18650 155257
169952 238214
165817 227895
157305 170792
95006 253933
99939 235475
194735 249131
145156 40356
78161 71267
282738 145334
121785 101468
164552 153673
190341 118248
198269 37536
92813 170998
27060 8399
219851 1877
43438 149436
242389 2158
243874 164303
88389 80142
83954 181533
170643 192664
34060 3718
9784 246166
286985 90380
239016 127611
72794 32705
81578 242487
162151 3904
217917 201835
41529 142717
104290 162556
65387 71072
138234 56394
61812 118226
249835 39647
247339 194938
78797 212928
106913 212978
45227 29155
27133 163079
181185 133728
82429 33658
43177 99866
189667 36693
93247 105301
9621 67165
209126 13699
138804 38213
245994 58829
151345 232080
74192 225427
280548 21258
242566 147117
31582 226998
247152 232513
27734 134512
15396 21607
246566 210471
208164 96989
78299 174738
117869 83846
14130 62081
49679 136406
23090 69710
105709 90147
23743 158061
274034 159
267659 186251
216209 55868
61450 124380
257279 129793
65877 238190
55181 241519
125205 126278
25341 163778
86276 17143
263226 212592
38956 82164
151449 97281
169998 97902
186624 104246
123358 227258
49798 235188
71605 192093
134803 129836
75543 169968
269851 237205
269038 217267
266537 253271
183605 120512
44873 36667
18010 26633
54650 163159
21368 33006
265192 85462
205229 50361
101636 60829
187283 214098
263592 112394
200660 81353
142451 145986
48449 121450
137168 57803
164166 208997
70940 111762
178085 180105
81474 166321
45996 133329
277015 93080
273446 58814
139263 9682
246795 61191
231864 156215
115092 153931
126910 86970
287069 62018
29726 230645
74835 184249
69866 107328
246557 135854
149154 28370
173168 218793
34469 162300
105687 150196
276217 86672
37473 128849
49681 13365
58712 70682
31167 55157
26500 115602
3797 228565
64078 220107
168520 40009
39954 23638
8018 18818
20594 238397
96492 58399
93618 157182
259084 248312
48516 203361
18449 179119
238347 143520
186866 13429
235779 185336
13483 40119
226531 36548
211010 224281
135865 140471
259317 203414
233601 56735
49437 131785
106662 62725
201860 110262
286955 12435
69478 76793
2746 61618
216014 184175
134553 67169
199963 32303
70360 89121
177805 67997
38206 208608
242523 207738
65157 136145
258521 203932
34614 150718
47570 141671
275146 179848
82110 98164
135855 210009
68493 68064
234214 101596
228476 200048
234860 185501
63808 57053
137731 224590
102165 37536
115455 140466
54899 126996
10373 74321
11388 86531
216337 218830
232198 179383
230139 50028
147653 128810
115391 206530
111230 237807
66689 190080
85658 115858
139271 127713
120007 230893
174276 46466
192292 41483
56926 6465
183080 247553
83873 174082
238299 10860
251038 199624
205445 123470
196098 15430
111085 163696
44682 147087
280352 19778
141737 61411
194789 58203
40845 146714
213757 241722
285141 109800
172218 171032
230732 18133
152203 19112
150207 126677
11677 142738
163574 85906
16539 13103
229820 14174
113628 253824
84293 59968
118219 132495
281938 210732
136767 206606
172665 251018
161335 117963
94310 112187
242077 255846
195129 207071
96155 186891
18364 151218
161441 221203
59555 179346
7591 217937
73528 200798
24144 17667
42507 151317
214332 65266
150733 68555
182184 108388
233556 140037
60311 96874
127893 187127
221612 61719
73188 163998
106675 172038
144970 79869
60774 101982
192509 248232
139293 215904
168856 155830
114459 106785
25426 4366
57119 18783
142020 73943
214722 37002
167645 2551
135887 165168
243804 50873
153395 70880
243854 96048
218922 86164
123584 107398
230477 232824
285384 203815
173626 121552
275525 218497
102411 109690
87299 252476
40808 107369
160801 159837
182880 27515
284979 121374
107854 43288
160771 211582
190239 33650
58974 244612
24793 93430
4622 255373
257359 115914
224766 15392
156695 231505
240445 75253
134168 75718
228 122379
225975 130025
15657 62194
164338 12625
240131 167478
285543 91726
3390 195337
263980 162056
115917 191554
158009 201513
220273 160623
278080 123625
279765 180207
74179 90474
95428 15349
214239 33139
140901 115486
178004 37000
11279 41445
69160 66690
25728 151012
281514 32571
255425 7972
37224 68218
219222 221088
279270 115625
44784 219557
61715 65242
83967 173202
21792 147565
146310 139244
126996 249763
20295 172937
118156 186058
39377 10347
234166 114966
155565 187895
107433 173861
180692 85446
282485 14715
172387 194129
185892 82541
104428 189840
283386 70725
5960 37442
144243 246967
61778 207445
259179 235989
10610 9132
70299 53870
218933 210026
72646 205263
62637 114531
211845 165419
204153 62348
115009 96996
240648 233065
57961 97200
120455 70862
190407 150372
141637 124957
21381 69644
160621 105414
13699 69304
23291 232053
194921 12216
92372 94982
98820 152945
206464 87582
222166 26878
73113 230156
40229 7893
164362 208631
83855 104297
219173 109237
249415 245588
227702 118339
234256 74304
58421 237293
34463 75176
162484 125813
261088 132042
234239 101579
92528 200425
112782 210532
235492 249156
129942 23243
266416 243985
57766 37888
16545 5827
194107 151992
63069 256061
201476 243310
272700 135332
100810 84410
286167 248743
232016 226213
147704 147964
104380 131271
38343 232948
146069 15796
262465 6442
127247 57551
146495 90652
197706 93990
67348 6717
138320 97132
35966 71588
242991 33005
196212 40134
8089 100180
2859 172412
40148 179673
243919 242723
62759 50686
282451 22914
235140 76605
134653 130078
23251 151256
57340 78735
61162 208402
30963 156081
183213 183073
23638 104947
111577 12493
185172 49337
5459 110505
173122 163510
20168 75394
84613 238135
179877 248231
147496 676
248231 25389
177406 231740
198498 157223
103023 202489
248693 41422
59791 90943
115852 31036
196822 114816
116362 116974
175848 185022
68513 42708
141586 241854
116700 220211
238343 208603
134317 133286
38699 14401
257382 113752
34372 171264
16375 227704
133317 138094
208296 139872
71856 225523
198322 213157
130439 30829
73003 142507
283956 142140
38720 220931
275658 243327
114038 131223
279567 245443
210069 120752
70465 212271
95272 116912
60342 225619
267883 213864
55577 145705
14460 33639
38434 177294
193312 150484
60879 57099
211520 120593
13434 70633
14135 96188
71964 97759
84674 195177
149637 82592
149101 179662
132235 80769
235316 176591
100100 235424
130966 147623
79757 104866
165537 21903
228689 162924
223705 233328
168404 241702
187704 13326
100888 255687
46762 254622
159995 61988
197190 119500
49125 110130
261391 75325
97596 158541
280506 34645
118286 216547
120392 11284
45476 13341
88336 234366
112932 63845
73 115433
168237 14721
