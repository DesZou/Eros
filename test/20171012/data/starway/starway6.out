4882.445954642
